support_matrix/index.md

```diff
@@ -0,0 +1,885 @@
+# Support Matrix
+
+## Supported platforms and features in {{ site.data.product.title_short }}
+
+A summary of the supported platforms and features in {{ site.data.product.title_short }}.
+{:shortdesc}
+
+- [Introduction](#introduction)
+- [Supported Browsers](#supported-browsers)
+- [Supported Platforms](#supported-platforms)
+- [Provisioning Support on Infrastructure Providers](#prov-infrastructure-providers)
+- [Provisioning Support on Cloud Providers](#prov-cloud-providers)
+- [Feature Support on Infrastructure Providers](#feature-infrastructure-providers)
+- [Feature Support on Cloud Providers](#feature-cloud-providers)
+- [Feature Support on Container Providers](#feature-container-providers)
+- [Feature Support on Network Providers](#feature-network-providers)
+- [Feature Support on Configuration Management Providers](#feature-configuration-management-providers)
+- [Remote Console Support](#remote-console-support)
+- [Operating System Support](#operating-system-support)
+- [Smart State Analysis Support](#smart-state-analysis-support)
+- [Supported Guest Operating Systems](#supported-guest-operating-systems)
+
+## Chapter 1. Introduction
+{: #introduction}
+This release of {{ site.data.product.title_short }} is available in 2 formats: virtual appliance and podified (via that includes the following key components:
+
+- CentOS Linux release 8.2.2004 (Core)
+- Rails 5.2.4.3
+- Ruby 2.6.3p62
+- PostgreSQL 10.6
+- Apache 2.4.37
+
+## Chapter 2. Supported Browsers
+{: #supported-browsers}
+The following table lists the browsers that work with {{ site.data.product.title_short }}.
+
+#### Table 2.1. Supported Browsers
```

Comment:
We don't use the Chapter / Table notation elsewhere in the docs.  The level is implied by the header notation.